README.md

```diff
@@ -1,2 +1,267 @@
+- [IPLD](https://github.com/ipld/roadmap/pull/1)
+
+## 2019 Epics
+
+We've coalesced the Working Group Milestones into 2019 IPFS Epics in order to create a overview of the primary targets the IPFS Project is setting for 2019. The information about Timeline, Sequencing and Priority still resides within each Working Group Roadmap.
+
+If you are pumped about these Epics and want to help, you can also get involved! See the call to action (CTA) for each section below.
+
+**1. The reference implementations of the IPFS Protocol (Go & JS) become Production Ready**
+
+We've been working hard on improving and iterating the IPFS Protocol to make sure it serves its users needs and achieves the goal of giving the Web a new super power, Content Addressing. In 2019, we are setting ourselves to solidify the core components and ensure IPFS is ready for Production Systems. Some of these goals include:
+- IPFS core solidified and APIs revamped/crystalized to be future-proof
+- Complete Specification of the Protocol
+- Security Audits and Security Program
+- Go and JS IPFS enable modern IPFS data formats (UnixFSv2, CIDv1, raw blocks) by default
+- Bring js-ipfs and go-ipfs to v1.0.0
+
+You can contribute to these Epic by testing, debugging or even coding to either Go or JS IPFS. Both teams have a chat every week that you can join. [Go](https://github.com/ipfs/team-mgmt/blob/master/MGMT_GOLANG_CORE_DEV.md) & [JS](https://github.com/ipfs/team-mgmt/blob/master/MGMT_JS_CORE_DEV.md#weekly-core-dev-team-calls).
+
+**2. Support Software Package Managers into entering the Distributed Web**
+
+Software Package Managers are one of the best systems to take advantage of Content Addressing. Developers and Deployment systems hit Package Managers Registries everyday to download the same bits of code, creating a huge overhead in terms of bandwidth. In addition today, it is incredibly important that we ensure that the code that we run is the code that the author's intended and Content Addressing can help with this goal by adding a Integrity verification step.
+- Kick-off a Package Managers Working Group to work closely with different Package Managers and understand each one challenges and needs.
+- Publish a report on the results from adding the NPM Registry to IPFS.
+- Meet the scalability and performance needs to serve the Package Managers users.
+
+We will booting the Package Managers Working Group soon, meanwhile, check out [NPM on IPFS](https://github.com/ipfs-shipyard/npm-on-ipfs) & [GX](https://github.com/whyrusleeping/gx) to get a feel for what IPFS can contribute to Package Management.
+
+**3. Scale the IPFS Network**
+
+To support the continued growth of the IPFS Network, we plan to optimize our scalability and testing.
+- Deploy a P2P testbed for Distributed Protocols that enables us to spin up hundreds of thousands of nodes on demand and run tests on them.
+- IPFS has an opt-in Telemetry system so that tests can collect key data to understand the experiments.
```

Comment:
hmm, I always figured this was more about us understanding how IPFS is growing. Opt-in sounds disabled by default (not what we want), but I think opt-out is not right as I'd like to think we'd confirm with people before just collecting and reporting them. Maybe "optional" is a better word?